docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -230,55 +230,6 @@ This review step is to ensure that all acceptance tests pass on the release bran
 
       export VERSION=0.1.2
```

Comment:
You don't really need this exported in this section, but then again it is useful(ish) for the link checking to know what $VERSION is meant to mean.